### PR TITLE
Improve styling of PieChart

### PR DESCRIPTION
**Pull request by someshkhandelia**
### Summary of changes

Improve styling of PieChart

### Context and reason for change

- Updated overall colors.
- Updated tooltip.

Fix: #1092

Signed-off-by: someshkhandelia <someshkhandelia@gmail.com>

### CODE REVIEW

**Comment by someshkhandelia** [comment]
### Previously
Screenshot 2022-10-19 at 11 38 10

### Now
After the recent changes, it looks like this:
(P.S. The font of the legend cannot be changed. There is no way to specify it in the 3rd party API call.)

image

**Comment by vasily-pozdnyakov** [comment]
In my opinion the colors are too bright. Also not sure that it is a good idea to use criticality colors (opossum.orange) for other pie charts.

File: src/Frontend/Components/PieChart/PieChart.tsx [hunk 1]
```diff
@@ -55,21 +66,25 @@ export function PieChart(props: PieChartProps): ReactElement {
...
+          <RcTooltip
+            contentStyle={classes.tooltipContentStyle}
+            itemStyle={classes.tooltipItemStyle}
+          />
           <RcLegend
             verticalAlign="middle"
             align="right"
             layout="vertical"
             width={250}
+            iconSize={5}
```
[the remark this answers is not in the document]
Comment (someshkhandelia) [reply src/Frontend/Components/PieChart/PieChart.tsx:87]: Thanks @alexzurbonsen. I missed that, will change.

**Comment by leslielazzarino** [comment]
The orange is almost inpossible to read on my screen, it should be replaced.

**Review by leslielazzarino — CHANGES_REQUESTED** [review]
Replace hard to read color.

**Answered by @someshkhandelia**: Fixed.

File: src/Frontend/Components/PieChart/PieChart.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexzurbonsen) [reply src/Frontend/Components/PieChart/PieChart.tsx:87]: Thank you :)

**Review by benedikt-richter — CHANGES_REQUESTED** [review]
Looks much more consistent with other components now 👍 
As Alex pointed out the font of the legend is to be changed

**Comment by someshkhandelia** [comment]
After the latest changes, the PieCharts look like this:

image

image

**Review by benedikt-richter — APPROVED** [review]

**Review by alexzurbonsen — APPROVED** [review]

**Review by leslielazzarino — APPROVED** [review]

**Pull request merged**